.github/workflows/rust.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           # Disabled due to issues with cross-rs
           #x86_64-pc-windows-gnu,
         ]
-        channel: [1.80.0, nightly]
+        channel: [1.84.0, nightly]
```

Comment:
Is there a reason why this was hard-coded to 1.80.0? Since we're going to have a higher MSRV, is there a reason why we shouldn't make this just use the MSRV determined from `Cargo.toml` instead?

Reply:
I believe some of our dev-dependencies had a higher MSRV than us, so we could only test on 1.80. MSRV is checked separately with cargo-msrv.